dataland-frontend/tests/component/components/pages/QualityAssurance.cy.ts

```diff
@@ -0,0 +1,100 @@
+import QualityAssurance from "@/components/pages/QualityAssurance.vue";
+import { minimalKeycloakMock } from "@ct/testUtils/Keycloak";
+import { FixtureData, getPreparedFixture } from "@sharedUtils/Fixtures";
+import { DataMetaInformation, DataTypeEnum, PathwaysToParisData, SmeData } from "@clients/backend";
+import { QaStatus } from "@clients/qaservice";
+import { assertDefined } from "@/utils/TypeScriptUtils";
+
+describe("Component tests for the Quality Assurance page", () => {
+  const keycloakMock = minimalKeycloakMock({
+    roles: ["ROLE_USER", "ROLE_UPLOADER", "ROLE_REVIEWER"],
+  });
+
+  /**
+   * Mock a data meta information object for a dataset specified by its fixture data, data Id and data type
+   * @param fixtureData The fixture data corresponding to review
+   * @param dataId The Id corresponding to the dataset to review
+   * @param dataType The type of the dataset to review
+   * @returns the data meta information object created
+   */
+  function mockQaDataObjectForDatasetWithId(
```

Comment:
The function name does not reflect that only mocked data meta information is returned. I find that "Qa" has no reason to be in the name.

Reply:
Of course, you're right. Probably I had a different idea about what the function is supposes to do initially and forget to adapt it afterwards. Changed it